src/material-experimental/mdc-slider/slider.ts

```diff
@@ -170,6 +164,24 @@ export class MatSlider implements AfterViewInit, OnDestroy {
     }
   }
 
+  /**
+   * Ensures that there is not an invalid configuration for the slider thumb inputs.
+   */
+  _validateInputs(): void {
+    if (this._isRange()) {
```

Comment:
These should be guarded with an `ngDevMode` check so we can remove the checking code in production.